<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeWidget</class>
 <widget class="QWidget" name="MeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 123, 255);</string>
   </property>
   <property name="text">
    <string>基本信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 123, 255);</string>
   </property>
   <property name="text">
    <string>联系方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 123, 255);</string>
   </property>
   <property name="text">
    <string>用户帮助</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>160</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border:1px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bio">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>261</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border:1px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>我的Bio：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border:1px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>261</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border:1px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>手机:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apply">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;立即保存你的修改&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:#13A4DB; 
	background-color:#F1FBFE;

	transition-duration: 0.4s;
    border: 3px;
    border-radius: 15px;
}

QPushButton:hover{
	background-color:#1383B6;
	color:white;
}

</string>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;不做任何保存，返回主界面&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:#DF3B88; 
	background-color:#FFEDF3;

	transition-duration: 0.4s;
    border: 3px;
    border-radius: 15px;
}

QPushButton:hover{
	background-color:#DF3B88;
	color:white;
}

</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;保存并返回主界面&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:#34C471; 
	background-color:#E9FCF1;

	transition-duration: 0.4s;
    border: 3px;
    border-radius: 15px;
}

QPushButton:hover{
	background-color:#34C471;
	color:white;
}

</string>
   </property>
   <property name="text">
    <string>好</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/hr_long.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/hr_long.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>60</y>
     <width>380</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #444444;</string>
   </property>
   <property name="text">
    <string>常见问题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Q1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#8590A6;
background:transparent;
text-align:left;
}
QPushButton:hover{
background-color:#E9FCF1;
color:black;
}</string>
   </property>
   <property name="text">
    <string>1.程序出现了问题？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Q2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#8590A6;
background:transparent;
text-align:left;
}
QPushButton:hover{
background-color:#E9FCF1;
color:black;
}</string>
   </property>
   <property name="text">
    <string>2.邮箱不可用？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Q3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#8590A6;
background:transparent;
text-align:left;
}
QPushButton:hover{
background-color:#E9FCF1;
color:black;
}</string>
   </property>
   <property name="text">
    <string>3.忘记密码怎么办？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Q4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#8590A6;
background:transparent;
text-align:left;
}
QPushButton:hover{
background-color:#E9FCF1;
color:black;
}</string>
   </property>
   <property name="text">
    <string>4.手机丢失，手机号不可用怎么办？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Q5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>380</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color:#8590A6;
background:transparent;
text-align:left;
}
QPushButton:hover{
background-color:#E9FCF1;
color:black;
}</string>
   </property>
   <property name="text">
    <string>5.我要绑定的手机号提示我已经注册了，怎么办？</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/hr_long.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 123, 255);</string>
   </property>
   <property name="text">
    <string>安全</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>221</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border:1px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_showPassword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;显示密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:#DF3B88; 
	background-color:#FFEDF3;

	transition-duration: 0.4s;
    border: 3px;
    border-radius: 15px;
}

QPushButton:hover{
	background-color:#DF3B88;
	color:white;
}
</string>
   </property>
   <property name="text">
    <string>🔑</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username_hint">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_hint">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
